modules/ROOT/pages/upgrade-self-managed.adoc

```diff
@@ -0,0 +1,90 @@
+= Upgrade Runtime Fabric for Self-Managed Kubernetes
+
+To upgrade Anypoint Runtime Fabric for Self-Managed Kubernetes, upgrade Runtime Fabric components and the `rtfctl` command-line utility.
+
+== About Component Upgrades
+
+Upgrading Runtime Fabric components means upgrading services that:
+- Facilitate communication to the control plane
+- Provide load balancing
+- Log and forward metrics
+
+When component upgrades are available, you apply the upgrades within Runtime Manager via a button that's visible when an upgrade is available.
+
+Runtime Fabric component upgrades do not require adding a node.
+
+=== Determine the Component Versions You Are Using
+
+From a local workstation that has a Kubeconfig file and can access Kubernetes, or from a support bastion, run the following command:
+```
+.rtfctl version
+```
+
+The output looks something like this:
+
+* The `rtfctl` command-line utility version in the `rtfctl` field
```

Comment:
These seem out of place. Shouldn't it be...The output looks something like this:
<code example>
Also, Kim's edit seems to say omit these two bullet points. ?

Reply:
@valkyrie69 thanks for catching this. I missed this one. I moved the bullets under the code sample